lua/tmux/navigate.lua

```diff
@@ -40,16 +40,13 @@ local function has_tmux_target(border)
         return false
     end
 
```

Comment:
just check here if tmux is in zoom and return true if so.

Thus, we can keep the functionality and guarantee that zoom will break if persist_zoom is disabled.